spring-rabbit/src/test/java/org/springframework/amqp/rabbit/connection/CachingConnectionFactoryTests.java

```diff
@@ -1111,4 +1112,28 @@ public void setAddressesTwoHosts() throws IOException {
 				aryEq(new Address[] { new Address("mq1"), new Address("mq2") }));
 		verifyNoMoreInteractions(mock);
 	}
+
+	@Test
+	public void testChannelCloseIndepotency() throws IOException {
```

Comment:
Typo in method name - also this test doesn't appear to be verifying the fix - it passes with the existing connection factory logic.